README.md

```diff
@@ -127,13 +156,15 @@ limitd.take(type, key, [count], (err, result) => {
 -  `key`: the identifier of the bucket.
 -  `count`: the amount of tokens you need. This is optional and the default is 1.
 -  `configOverride`: caller-provided bucket configuration for this operation
+-  `erlIsActiveKey`: (string) the identifier of the ERL activation for the bucket. Only mandatory to pass while operating on buckets that have ERL configured.
+-  `allowERL`: (boolean) optional. if set to true, the ERL feature will be allowed to be used.
```

Comment:
I wonder if this is required or can be completely omitted in favor of a more clean approach.
In general, we try to avoid changing method signatures and use bucket definitions to give higher configuration flexibility and compatibility with the clients.

If possible, I propose using `configOverride` for setting `allowERL` in a given operation (we have similar cases using it).

For `erlIsActiveKey`, I propose moving it to top level LimitD configuration as it is not expected to change, and can be used as as prefix to generate dynamic keys per bucket definition if required.

Reply:
`allowERL` is meant to be used for passing down the value of a feature flag, which we want to use as a breaking glass mechanism to disable the whole ERL logic completely. 

Regarding `erlIsActiveKey`, we mean to use the tenant name with a suffix for it, as ERL will be configured per tenant. So the value of it has to come from outside and is not static.

Reply:
`allowERL` is removed with the most recent changes by @pubalokta